src/modules/translation/languages/en.json

```
@@ -1366,7 +1366,11 @@
     "add_item": "Add Item",
     "limit_reached": "You can only mint {max} items per transaction",
     "no_items": "You have no items available to mint for \"{name}\"",
-    "unsynced_warning": "Some items have changes that have not been approved yet."
+    "unsynced_warning": "Some items have changes that have not been approved yet.",
+    "next": "Next",
+    "total_items": "Total items to be minted",
+    "confirm_title":  "You are about to mint {amount} items, each item will be minted to the given address with the provided amount.",
```

There's a way to write different cases of plurality differently, for example, if there's one item to be minted, we can write "item" and when there are several, we can write "items":
```suggestion
    "confirm_title":  "You are about to mint {amount} {count, plural, one {item} other {items}}, each item will be minted to the given address with the provided amount.",
```